public/editor/scripts/project/project.js

```diff
@@ -244,15 +244,21 @@ define(function(require) {
               return callback(err);
             }
 
-            // Look for an HTML file to open, ideally index.html
-            var indexPos = 0;
+            // Look for an HTML file to open
```

Comment:
There's another code change happening in https://github.com/mozilla/thimble.mozilla.org/pull/1822 you should be aware of, and integrate with, once it lands.